slatedb/src/db.rs

```diff
@@ -3144,6 +3147,70 @@ mod tests {
         assert_eq!(db.inner.mono_clock.last_tick.load(Ordering::SeqCst), 11);
     }
 
+    #[tokio::test]
+    async fn test_put_get_reopen_delete_with_separate_wal_store() {
+        let object_store: Arc<dyn ObjectStore> = Arc::new(InMemory::new());
```

Comment:
you don’t have to declare the type of this variable, this works just fine:

```suggestion
        let object_store = Arc::new(InMemory::new());
```

Same applies below, and in the other tests. I’m guessing this is a copy paste from other places because I removed a bunch of them in my last PR 😅

Reply:
fixed by https://github.com/slatedb/slatedb/pull/558/commits/12e6d8f42ce6ace7dd65b3739944b1ee414f27ed